pkg/quota/evaluator/core/pods.go

```diff
@@ -269,18 +258,18 @@ func PodUsageFunc(obj runtime.Object, clock clock.Clock) (api.ResourceList, erro
 	if err != nil {
 		return api.ResourceList{}, err
 	}
-	// by convention, we do not quota pods that have reached end-of life
-	if !QuotaPod(pod, clock) {
-		return api.ResourceList{}, nil
+
+	// always quota the object count (even if the pod is end of life)
```

Comment:
Explain the change in more detail please.  Why now?

Reply:
added more text.

"pods" == pods that are not in a terminal state (i.e. phase is not Succeeded or Failed)
"count/pods" == all the pods independent of state.